.github/workflows/generator.yml

```diff
@@ -11,7 +11,7 @@ env:
 
 jobs:
   Build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Why not 22.04?

Reply:
22.04 Has Python 3.10, and our current unit test framework (nose) does not work with Python 3.10 :-(